frontend/components/site/siteBuildLogs.jsx

```diff
@@ -26,11 +52,19 @@ const SiteBuildLogs = () => {
           <li><DownloadBuildLogsButton buildId={buildId} buildLogsData={logs} /></li>
         </ul>
       </div>
-      {(!logs || logs?.length === 0) && (
-      <div>
-        <SiteBuildLogTable buildLogs={['This build does not have any build logs.']} />
-      </div>
-      )}
+      {(() => {
+        if (!logs || logs?.length === 0) {
```

Comment:
Is there a scenario where there are build logs and the build is over 180 days?

Reply:
Yes, the scenario you found on dev where the build log was still being shown on an old build that still had build logs: https://pages-dev.cloud.gov/sites/2/builds/5/logs

Reply:
That's why we made this change.

Reply:
The logic still shows both the 180 day message and the build logs.

Reply:
It only checks `buildTooOld` if there are no build logs.